components/brave_ads/browser/bundle_state_database.cc

```diff
@@ -222,10 +222,10 @@ bool BundleStateDatabase::InsertOrUpdateCreativeAdNotification(
     statement.BindString(5, info.end_at_timestamp);
     statement.BindString(6, info.creative_instance_id);
     statement.BindString(7, info.campaign_id);
-    statement.BindInt(8, info.daily_cap);
+    statement.BindInt(8, static_cast<int>(info.daily_cap));
```

Comment:
use base/numerics/safe_conversions for unsigned -> signed int

Reply:
I don't think we want checked_cast or saturated_cast here. Perhaps it's better to use `statement.BindInt64` instead?

Reply:
@bridiver : Switched to using BindInt64 in f9d3dfd723fa08bdc8c686ef3130f6ec8791222a